src/plugins/seeSummaries/index.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Vencord, a Discord client mod
+ * Copyright (c) 2024 Vendicated and contributors
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+import { DataStore } from "@api/index";
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+import { FluxDispatcher } from "@webpack/common";
+
+
+export default definePlugin({
+    name: "Summaries",
+    description: "Enables summaries and persists them on restart",
+    authors: [Devs.mantikafasi],
+    patches: [
+        {
+            find: "ChannelTypesSets.SUMMARIZEABLE.has",
+            replacement: {
+                match: /\(.\){var .;let .*GuildFeatures.SUMMARIES_ENABLED_BY_USER\)\)}/,
+                replace: "(e){return true;}"
```

Comment:
untested but probably works, way cleaner and safer. just overrides the boolean to always be true
```suggestion
                match: /\i\.hasFeature\(\i\.GuildFeatures\.SUMMARIES_ENABLED_BY_USER\)/,
                replace: "true"
```